apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -129,6 +129,15 @@ func (r *OpenTelemetryCollector) validateCRDSpec() error {
 			return fmt.Errorf("the OpenTelemetry Spec autoscale configuration is incorrect, minReplicas should be one or more")
 		}
 
+		if r.Spec.Autoscaler != nil {
```

Comment:
We should also check that min or max replicas is set in order to use this feature

Reply:
That's inside an if statement that's checking whether maxReplicas is set.